drivers/storage/portworx/deployment.go

```diff
@@ -287,7 +287,7 @@ func (p *portworx) GetStoragePodSpec(
 		}
 	}
 
-	if pxutil.IsTelemetryEnabled(cluster.Spec) {
+	if pxutil.IsTelemetryEnabled(cluster.Spec) && !pxutil.RunCCMGo(cluster) {
```

Comment:
nit: use new and old ccm instead of go and java, easier to understand as other people may not know what language it's written

Reply:
IMO `CCMGo` is better than calling old/new because if we later have another version of ccm, it will be get confusing to name it. Have some logical name is better.

Reply:
keeping go and java